tdrs-backend/tdpservice/parsers/schema_defs/cat3_validators.py

```diff
@@ -0,0 +1,137 @@
+"""Collection of category three validators for TANF Section 1 records."""
+
+
+from .. import validators
+
+
+def validate_disposition(record):
+    """Validate that dispotion and it's dependent fields."""
+    disposition = getattr(record, "DISPOSITION")
+    if disposition == 2:
```

Comment:
Define a static showing what 2 is referring to: e.g: MAX_ALLOWED_DISPOSITION = 2